test-utils/xhrMock.js

```diff
@@ -83,8 +82,8 @@ module.exports = function() {
 		$defineRoutes: function(rules) {
 			routes = rules
 		},
-		$defineJSONPCallbackKey: function(key) {
-			callback = key
+		$defineJSONPCallbackKey: function(/* key */) {
```

Comment:
I assume because `callback` never gets called, which might be reported by ESLint, but @isiahmeadows missed to remove it ;)